.gitignore

```diff
@@ -27,3 +27,7 @@
 
 # system
 .DS_Store
+
+# Custom
+
+**/.gitkeep
```

Comment:
Non ! il ne faut pas ignorer les .gitkeep. C'est une [convention adoptées par les utilisateurs de Git](http://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep), et qui permet de commiter des arborescences ([chose impossible dans le fonctionnement de Git](https://git.wiki.kernel.org/index.php/Git_FAQ#Can_I_add_empty_directories.3F)).